Source/Extensions/Blazorise.DataGrid/_DataGridAggregateRow.razor

```diff
@@ -20,7 +20,7 @@
         }
         else
         {
-            <TableRowCell Class="@column.GroupCellClass" Style="@column.BuildGroupCellStyle()" TextAlignment="@column.TextAlignment" VerticalAlignment="@column.VerticalAlignment" Display="@column.Display" Flex="@column.Flex" Gap="@column.Gap">
+            <TableRowCell Class="@column.GroupCellClass" Style="@column.BuildGroupCellStyle()" TextAlignment="@column.GroupCellTextAlignment" VerticalAlignment="@column.GroupCellVerticalAlignment" Display="@column.GroupCellDisplay" Flex="@column.GroupCellFlex" Gap="@column.GroupCellGap">

```

Comment:
So the **Group** prefix is to be applied on **Aggregates**? I think it can be mistaken with the new group feature we are working on, isn't it more clear to straight up use the **Aggregate** prefix?

Reply:
You're right. I forgot about the new group feature. I will change it.

Reply:
Done. Renamed to Aggregate. And made existing GroupCellClass and GroupCellStyle obsolete.